src/core/messages.ts

```diff
@@ -26,6 +26,26 @@ import EventEmitter from './utils/event-emitter.js';
  */
 interface MessageEventsMap {
   [MessageEvents.Created]: MessageEventPayload;
+  [MessageEvents.Edited]: MessageEventPayload;
```

Comment:
```suggestion
  [MessageEvents.Updated]: MessageEventPayload;
```
? Not sure if we're going with `update` everywhere?

Reply:
Yeh we should, I've made the change